proto/tendermint/abci/types.proto

```diff
@@ -300,18 +314,19 @@ message ResponseApplySnapshotChunk {
   }
 }
 
+// ResponsePrepareProposal reflects the same protobuf message as CometBFT
+// The exception here is that the first transaction is always the hash
```

Comment:
[question] Based on code above, I thought the second to last transaction was the Merkle root of the row + col roots and the last transaction was the square size. If so, is this comment inaccurate?

Reply:
Good catch, I forgot to update the comments when I changed it